ABloom/ABloom/Presentation/CoreViews/QnA/View/AnswerWriteView.swift

```diff
@@ -57,17 +69,49 @@ struct AnswerWriteView: View {
 extension AnswerWriteView {
   
   private var questionText: some View {
+    
     HStack(alignment: .top, spacing: 11) {
-      Image("avatar_Female circle GradientBG")
+      
+      // 유저가 남성일 때,
+      //      Image("avatar_Female circle GradientBG")
+      //        .resizable()
+      //        .frame(width: 34, height: 34)
+      //
+      //      VStack {
+      //        LeftPinkChatBubble(text: question.content)
+      //        LeftPinkChatBubble(text: "너의 생각을 알려줘")
+      //      }
+      
+      
+      // 유저가 여성일 떄,
+      Image("avatar_Male circle GradientBG")
         .resizable()
         .frame(width: 34, height: 34)
       
       VStack {
-        LeftPinkChatBubble(text: question.content)
-        LeftPinkChatBubble(text: "너의 생각을 알려줘")
+        LeftBlueChatBubble(text: question.content)
+        LeftBlueChatBubble(text: "너의 생각을 알려줘")
       }
+      
       Spacer()
     }
     .padding(.horizontal, 22)
   }
+  
+  private var answerText: some View {
+    HStack {
+      Spacer()
+      
+      // 유저가 남성일 때,
+      // BlueChatBubbleTextField(text: $answerVM.answerText)
+      
+      // 유저가 여성일 떄,
+      PinkChatBubbleTextField(text: $answerVM.answerText)
+      
+        .onChange(of: answerVM.answerText, perform: { newValue in
+          answerVM.answerText = String(newValue.prefix(150))
```

Comment:
```suggestion
        if newValue.count > 150 {
            answerVM.answerText = String(newValue.prefix(150))
        }
```
지금 방식은 글자가 바뀔때마다 수행되는 거여서 수정되어도 좋을 것 같슴니당!

Reply:
아하! 다음 적용하겠습니다 :) 굳!